<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewMesGiag</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #e4f2f9;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f9f9f9; border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: none; color: #313131; font-size: 16px; text-align: left center; padding-left: 10px; border-bottom: 2px solid #313131; background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dataLine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: none; color: #313131; font-size: 16px; text-align: left center; padding-left: 10px; border-bottom: 2px solid #313131;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeLine">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: none; color: #313131; font-size: 16px; text-align: left center; padding-left: 10px; border-bottom: 2px solid #313131;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Время</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>312</width>
      <height>201</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCalendarWidget QToolButton{
color: #313131;
padding: 0 10px;
font-weight: 400;
icon-size: 30px;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
    <property name="Visible" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: none; color: #313131; font-size: 16px; text-align: left center; padding-left: 10px; border-bottom: 2px solid #313131;</string>
    </property>
    <item>
     <property name="text">
      <string>Выберите услугу</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="phoneLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: none; color: #313131; font-size: 16px; text-align: left center; padding-left: 10px; border-bottom: 2px solid #313131; background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Телефон</string>
    </property>
   </widget>
   <zorder>comboBox</zorder>
   <zorder>nameLine</zorder>
   <zorder>dataLine</zorder>
   <zorder>timeLine</zorder>
   <zorder>dataBtn</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>phoneLine</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
